matrix/Quaternion.hpp

```diff
@@ -120,6 +121,69 @@ class Quaternion : public Vector<Type, 4>
         v(3) = w(2,0);
         return Q * v * Type(0.5);
     }
+
+    void invert() {
```

Comment:
maybe we should clearly separate the conjugate and the inverse
http://www.3dgep.com/understanding-quaternions/#Quaternion_Conjugate and http://www.3dgep.com/understanding-quaternions/#Quaternion_Inverse